app/views/hosts/console/vnc.html.erb

```diff
@@ -13,11 +13,9 @@
           {:id => "troubleshootingButton", :class => "btn btn-default"}
         )
  ) %>
-<p class="toggle-hidden-value"><%= _('Password:') %> <a href="#" class="toggle-hidden-value"><%=icon_text("plus")%></a><span><%= @console[:password] %></span></p>
```

Comment:
Is this removal on purpose? doesn't seem related to the layout change

Reply:
It was done on purpose, but after reading a bit on the PR that added this, I think the best course of action is to only show it if there ia a VNC password available. I will change this PR to reflect this.
The reason why I removed it, was that I saw "Password: " but the password, in my case, is always empty... which was short-sighted  from my side

Reply:
I suggest separating that change to a different pr in that case :)

Reply:
Okay, makes sense 👍 I'll remove this change. Thanks for the input